src/journal/journal-file.c

```diff
@@ -3517,7 +3534,7 @@ int journal_file_open_reliably(
         size_t l;
         _cleanup_free_ char *p = NULL;
 
-        r = journal_file_open(-1, fname, flags, mode, compress, seal, metrics, mmap_cache, deferred_closes, template, ret);
+        r = journal_file_open(-1, fname, flags, mode, compress, -1, seal, metrics, mmap_cache, deferred_closes, template, ret);
```

Comment:
Why do you set `-1` here instead of `compress_threshold_bytes`?

Reply:
This was a bug, I'm afraid